mensagemDaDigix.js

```diff
@@ -22,7 +22,20 @@ class MensagemDaDigix {
   executar() {
     this.removerStorageExpirado();
     let exibiuMensagem = this.obterLocalStorage();
+    
     if (!exibiuMensagem.value) {
+      const sistemas = this.sistemas;
+
+      if (sistemas.length > 0) {
```

Comment:
Nesse `forEach`, se existir algum sistema listado no array de sistemas, ele vai construir os elementos e inserir o local Storage pra esse sistema caso seja o do`window.location.host` atual. Porém, na linha 39, ele sempre vai rodar de qualquer forma o `construirElementos()` e o `inserirLocalStorage()`, independente do array de sistemas, está correto isso?

Reply:
Na linha 36 o `return` garante que, se a condição `sistemas.length > 0` for atendida, a execução do método `executar()` encerra.

Reply:
Ah, não tinha visto esse return. Essa análise de PR do GitHub é diferente